### PR TITLE
🔖 V0.1.0

### PR DESCRIPTION
**Pull request by RenardFute**
First release of the Engine
CleanShot 2024-10-31 at 06 10 56@2x

### CODE REVIEW

File: conanfile.py
Comment (Marius-P1) [inline conanfile.py:1]: Does that replace the conanfile.txt ?

File: test-release.sh
Comment (Marius-P1) [inline test-release.sh:1]: If that only a test for the release and not the final version you can delete that

**Review by Marius-P1 — APPROVED** [review]
Great :ok_hand:

File: conanfile.py
Comment (RenardFute) [reply conanfile.py:1]: Yes it does, it's more flexible and doesn't cause all the bugs I got somehow.

File: test-release.sh
Comment (RenardFute) [reply test-release.sh:1]: It's a script that emulates what teamcity does but it skips the upload part. So by running this anyone can "test" the release before merging/uploading

**Pull request merged**